recipes/annual-energy-use/annual_energy_use/_handler.py

```diff
@@ -0,0 +1,36 @@
+from queenbee_dsl.alias import InputAlias, OutputAlias
+from queenbee.io.common import IOAliasHandler
+
+
+input_model_alias = [
+    # grasshopper Alias
+    InputAlias.any(
+        name='model',
+        description='A path to a HBJSON file or a HB model object built with Python or'
+        'dotnet libraries.',
+        platform=['grasshopper'],
+        handler=[
+            IOAliasHandler(
+                language='python', module='honeybee_radiance_handlers.handlers',
+                function='model_to_json_path'
+            ),
```

Comment:
These names has changed. I should fix these for daylight-factor recipe.

https://github.com/pollination/handlers-python/blob/master/pollination_handlers/inputs/model.py

```suggestion
            IOAliasHandler(
                language='python', module='pollination_handlers.inputs.model',
                function='model_to_json'
            ),
```